configuration/exodus-pipeline.yaml

```diff
@@ -273,14 +284,14 @@ Resources:
   CodePipelineCloudTrail:
     Type: AWS::CloudTrail::Trail
     # Create trail for prod pipeline
-    Condition: IsProd
+    Condition: IsExodusProd
     Properties:
-      S3BucketName: exodus-pipeline-artifacts
+      S3BucketName: !Sub cloudtrail-logs
       EventSelectors:
         - DataResources:
           - Type: AWS::S3::Object
             Values:
-              - arn:aws:s3:::exodus-pipeline-artifacts/build-stage
+              - "arn:aws:s3:::"
```

Comment:
What's the intention here?

Remember that prod pipeline is triggered by changes to this object. So, I doubt the trail can be shared by multiple pipeline sets.